src/Microsoft.Health.Api/Registration/ApplicationBuilderExtensions.cs

```diff
@@ -36,6 +36,8 @@ public static void UseHealthChecksExtension(this IApplicationBuilder app, string
                                 name = entry.Key,
                                 status = Enum.GetName(typeof(HealthStatus), entry.Value.Status),
                                 description = entry.Value.Description,
+                                data = entry.Value.Data,
+                                exception = entry.Value.Exception,
```

Comment:
Perhaps exposing the exception is too much information, so we should continue to filter that out.